docs/en/latest/plugins/prometheus.md

```diff
@@ -46,17 +46,16 @@ plugin_attr:
 
 Assume environment variable `INTRANET_IP` is `172.1.1.1`, now APISIX will export the metrics via `172.1.1.1:9092`.
 
-**Before version `2.6`, the metrics are exposed via the data panel port,
-you may need to use [interceptors](../plugin-interceptors.md) to protect it.**
-
-If you still want this behavior, you can configure it like this:
+If you still want expose the metrics via the data panel port, you can configure it like this:
```

Comment:
```suggestion
If you still want to expose the metrics via the data panel port, you can configure it like this:
```

Reply:
@juzhiyuan Do you mean to set the `enable_export_server` default configuration to occur false, expose it to the DP port by default, and allow the user to set the route for this API?

Reply:
Because we don't use `data plane` in docs frequently, how about using `via the data plane port (default: xxx)` here?

Also, it should be `plane` instead of `panel`

Reply:
I think it is ok, I will add the description of the default port. The `panel` is a spelling mistake.

Reply:
@juzhiyuan updated